web-ts/src/routes/SprungbrettOfferPage.tsx

```diff
@@ -12,25 +38,82 @@ type PropsType = {
   languageModel: LanguageModel
 } & RouteComponentProps<{ cityCode: string; languageCode: string }>
 
-const SprungbrettOfferPage = ({ cityModel, match, location }: PropsType): ReactElement => {
-  const { languageCode } = match.params
+const SprungbrettOfferPage = ({ cityModel, match, location, languages }: PropsType): ReactElement => {
+  const { cityCode, languageCode } = match.params
+  const pathname = normalizePath(location.pathname)
+  const { viewportSmall } = useWindowDimensions()
+  const { t } = useTranslation('sprungbrett')
+
+  const requestOffers = useCallback(async () => {
+    return createOffersEndpoint(cmsApiBaseUrl).request({ city: cityCode, language: languageCode })
+  }, [cityCode, languageCode])
+  const { data: offers, loading: offersLoading, error: offersError } = useLoadFromEndpoint(requestOffers)
+
+  const offer = offers?.find(offer => offer.alias === 'sprungbrett')
+
+  const requestSprungbrettOffer = useCallback(async () => {
+    if (!offer) {
+      return new Payload(false, null, [])
+    }
+    return createSprungbrettJobsEndpoint(offer.path).request()
+  }, [offer])
+
+  const { data: sprungbrettJobs, loading: sprungbrettLoading, error: sprungbrettError } = useLoadFromEndpoint(
+    requestSprungbrettOffer
+  )
 
   const toolbar = (openFeedback: (rating: FeedbackRatingType) => void) => (
-    <LocationToolbar openFeedbackModal={openFeedback} viewportSmall={false} />
+    <LocationToolbar openFeedbackModal={openFeedback} viewportSmall={viewportSmall} />
+  )
+
+  const languageChangePaths = languages.map(({ code, name }) => {
+    return {
+      path: createPath(SPRUNGBRETT_OFFER_ROUTE, { cityCode, languageCode: code }),
+      name,
+      code
+    }
+  })
+
+  const locationLayoutParams = {
+    cityModel,
+    viewportSmall,
+    feedbackTargetInformation: offer ? { path: offer.path } : null,
+    languageChangePaths,
+    route: SPRUNGBRETT_OFFER_ROUTE,
+    languageCode,
+    pathname,
+    toolbar
+  }
+
+  if (offersLoading || sprungbrettLoading) {
+    return (
+      <LocationLayout isLoading {...locationLayoutParams}>
+        <LoadingSpinner />
+      </LocationLayout>
+    )
+  }
+
+  if (!sprungbrettJobs || !offer) {
+    const error = sprungbrettError || offersError || new Error('Sprungbrett and Offers should be available!')
```

Comment:
I think there should be an offer not found error if the sprungbrett offer is not enabled.